package.json

```diff
@@ -18,6 +18,10 @@
     "dev": "turbo run dev --parallel --env-mode=loose --filter=!n8n-design-system --filter=!@n8n/chat --filter=!@n8n/task-runner",
     "dev:be": "turbo run dev --parallel --env-mode=loose --filter=!n8n-design-system --filter=!@n8n/chat --filter=!@n8n/task-runner --filter=!n8n-editor-ui",
     "dev:ai": "turbo run dev --parallel --env-mode=loose --filter=@n8n/nodes-langchain --filter=n8n --filter=n8n-core",
+    "dev:fe": "run-p start \"dev:fe:turbo --filter=n8n-design-system\"",
+    "dev:fe:turbo": "turbo run dev --parallel --env-mode=loose --filter=n8n-editor-ui",
```

Comment:
~~Is `dev:fe` for editor-ui + design-system, and `dev:fe:turbo` only editor-ui? Maybe it could be called `dev:fe:editor`~~?

E: Oh I see it now. `dev:fe` starts BE, FE dev server and design system. `turbo` only FE dev server. Maybe it could be called `dev:fe:editor` then for clarity?

Reply:
Makes sense. Updated!